pages/settings/wallets/ConnectWallet.tsx

```diff
@@ -0,0 +1,281 @@
+              <Image
+                source={{ uri: appicon }}
+                className="my-4 rounded-2xl shadow-md w-20 h-20"
+              />
+            </View>
+            <Text className="text-xl font-semibold2">{appname}</Text>
+          </View>
+        </View>
+        {nwcUrl ? (
+          <View className="flex-1 w-full justify-center items-center">
+            <Tick />
+            <Text className="my-4 text-lg text-center text-foreground">
+              Connected! You're being redirected in {redirectCountdown}{" "}
+              seconds...
+            </Text>
+          </View>
+        ) : (
+          <>
+            <Text className="text-xl text-center text-foreground">
+              <Text className="text-xl font-semibold2">{appname}</Text> is
+              requesting to access your wallet{" "}
+              <Text className="text-xl font-semibold2">Alby Hub</Text> for
+              in-app payments.
+            </Text>
+            <View className="flex-1 flex mt-4 gap-8 justify-center">
+              <View>
+                {/* TODO: REPLACE WITH A PROPER COMPONENT */}
+                <TouchableOpacity className="flex flex-row border-2 border-muted justify-between items-center rounded-2xl p-6 py-4">
+                  <Text className="text-xl font-medium2">Monthly budget</Text>
+                  <View>
+                    <Text className="text-right text-lg text-foreground font-medium2">
+                      {new Intl.NumberFormat().format(+100_000) + " sats"}
+                    </Text>
+                    {getFiatAmount && (
+                      <Text className="text-right text-sm text-muted-foreground">
+                        {getFiatAmount(+100_000)}
+                      </Text>
+                    )}
+                  </View>
+                  {/* <Link href="/" className="absolute right-0" asChild>
+                    <TouchableOpacity className="p-4">
+                      <ChevronRightIcon
+                        className="text-muted-foreground"
+                        width={24}
+                        height={24}
+                      />
+                    </TouchableOpacity>
+                  </Link> */}
+                </TouchableOpacity>
+                <Text className="mt-4 text-center text-foreground">
+                  You can edit permissions and revoke access at any time in your
+                  Alby Hub settings.
+                </Text>
+              </View>
+            </View>
+          </>
+        )}
+      </View>
+      {!nwcUrl && (
+        <View className="p-6">
+          <View className="flex flex-row items-center justify-center gap-2 mb-4 px-4">
+            <WalletIcon className="text-muted-foreground" />
+            <Text
+              numberOfLines={1}
+              ellipsizeMode="tail"
+              className="text-muted-foreground font-medium2 text-xl"
+            >
+              {wallets[selectedWalletId].name || DEFAULT_WALLET_NAME}
```

Comment:
Can this be changable like other screens? I had the wrong wallet selected which didn't have `create_connection` scope.

Reply:
This is not changeable across the app, this is just to show the current wallet. Will open an issue for this one separately

Reply:
https://github.com/getAlby/go/issues/281